arch/src/lib.rs

```diff
@@ -19,6 +19,8 @@ extern crate hypervisor;
 extern crate kvm_bindings;
 extern crate kvm_ioctls;
 extern crate libc;
+#[macro_use]
+extern crate log;
 
 extern crate vm_memory;
```

Comment:
Could you delete this whitespace so they get reordered correctly.

Reply:
Yes!